pkgs/applications/misc/cool-retro-term/default.nix

```
@@ -1,5 +1,5 @@
 { stdenv, fetchFromGitHub, qtbase, qtquick1, qmltermwidget,
-qtquickcontrols, qtgraphicaleffects, qmake }:
+qtquickcontrols, qtgraphicaleffects, qmake, makeWrapper, qtdeclarative }:
```

```suggestion
, qtquickcontrols, qtgraphicaleffects, qmake, makeWrapper, qtdeclarative }:
```
+ remove the trailing comma in the line above.
It's the usual way of presenting it in nixpkgs